packages/kbn-performance-testing-dataset-extractor/src/extractor.ts

```diff
@@ -75,15 +82,16 @@ const getTraceItems = (
       return {
         processor: hit.processor,
         traceId: hit.trace.id,
-        timestamp: hit['@timestamp'],
+        date: hit['@timestamp'],
         environment: hit.environment,
         request: {
-          url: { path: hit.url.path },
-          headers: hit.http.request.headers,
+          path: hit.url.path,
+          headers: parseHeaders(hit.http.request.headers),
           method: hit.http.request.method,
-          body: payload ? parsePayload(payload, hit.trace.id, log) : undefined,
+          body: payload ? JSON.stringify(parsePayload(payload, hit.trace.id, log)) : undefined,
+          statusCode: hit.http.response.status_code,
+          timestamp: new Date(hit['@timestamp']).getTime(),
```

Comment:
Yes, it is the same. The idea was to keep everything Gatling needs under `request` key, but date is actually easier to use to debug journey traces

Reply:
Ok, let's keep it like that.

Reply:
As we discussed, I reverted `timestamp` to be date string and removed `date` property in 766bab3e23ee384ff753fea4333d8f3092d49b25. We will convert date string to timestamp in Gatling.
I also added `parentId` as it might be a way to identify concurrent transactions/filter out redirects on Gatling side.